casepro/backend/__init__.py

```diff
@@ -84,15 +84,12 @@ def create_label(self, org, name):
         """
 
     @abstractmethod
-    def create_outgoing(self, org, text, contacts, urns):
+    def push_outgoing(self, org, outgoing):
```

Comment:
Previously the pattern for creating something that exists in both RapidPro and CasePro, was 

1. Use an API call to create it in RapidPro
2. Get the identifier used by RapidPro
3. Create locally with that identifier

But for things that aren't being pushed to the backend - this pattern makes more sense:

1. Create locally
2. Optionally push to backend and update with returned identifier

Another advantage of this is that the local outgoing message is still created even if the RapidPro API call fails.